src/kaleidoscope/device/dygma/raise/wire-protocol-constants.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
```

Comment:
This is going to need to get a guard to only get compiled on the raise, I think. These constants and keyboardio's for the similar, but silghtly different wire protocol are going to stomp on each other.